Makefile

```diff
@@ -365,7 +365,7 @@ generate-manifests-core: $(CONTROLLER_GEN) $(KUSTOMIZE)
 	$(CONTROLLER_GEN) \
 		paths=./api/... \
 		paths=./controllers/... \
-		paths=./webhooks/... \
+		paths=./webhooks/internal/webhooks/... \
```

Comment:
I think we should move the webhooks to ./internal/webhooks

Reply:
Agreed - it's definitely possible we'll want to use some of the methods in the code base outside of the webhooks package.

Reply:
Not sure about this.
if there are methods that we want to use outside of the webhooks package, would they go into check package (or in a separate package) vs moving uplevel the entire webhook implementation?

Reply:
I'm okay either way - with the ClusterClass webhooks as they exist we're making the actual webhook implementation very thin to move things into the check package. 

But are we planning to do that with other webhooks? Presumably we're looking to 'eventually' include all webhooks in the top level package.

Reply:
My main motivation was to follow the pattern we established with our controllers. And we moved them all to `<provider-root-dir>/internal/controllers` not `<provider-root-dir>/controllers/internal/controllers`.

Reply:
> My main motivation was to follow the pattern we established with our controllers.

Ok let's stick to this model.
the target should be ./internal/webhooks instead of /webhooks/internal/webhooks/